weave/trace_server/costs/insert_costs.py

```diff
@@ -148,7 +148,8 @@ def sum_costs(data: Dict[str, List[CostDetails]]) -> float:
     return total_costs
 
 
-def insert_costs(client: Client) -> None:
+def insert_costs(client: Client, target_db: str) -> None:
+    client.database = target_db
```

Comment:
this matches existing pattern, but would be a bit odd if the same client was used in different target_dbs. No action, just calling out